### DISCUSSION TITLE
Adding in your documentation if vue can be installed with Alpine

### DISCUSSION THREAD

**Discussion by Benoit1980** [alpinejs/alpine#2638] [4. General]
Hello,

Would it be possible to add in your documentation if Vue js can be installed together with Alpine please?
I am trying to find out and all the posts I see online point to conflicts between the 2 but date from 2020, perhaps things have changed since?

Thanks

**Comment by xieyuheng** [comment]
Notes in this issues might help.

https://github.com/alpinejs/alpine/issues/363
